### ISSUE TITLE
Autostart of applications

### ISSUE THREAD

**Issue by dakyskye** [avahe-kellenberger/nimdow#35]
Add an option in config, to autostart apps on nimdow startup.  I don't know TOML much, but I guess that gonna work:
```toml
[autostart]
exec = [
    'firefox',
    'compton -cC',
    'sxhkd',
]
```

**Comment by avahe-kellenberger** [comment]
Pretty easy to add, but I'm curious if I should notify the user that one or more of the startup commands failed (other than logging or printing to stdout). Maybe notify-send - Thoughts?

**Comment by dakyskye** [comment]
If a program fails to start, do log it in `~/.xsession-errors`.

**Comment by dakyskye** [comment]
Any update?

**Comment by avahe-kellenberger** [comment]
Not yet, been focusing on floating window support for the beta release. It would be pretty simple to add though if you want to submit a PR before I can get to it (maybe late next week)

**Comment by dakyskye** [comment]
I might, stay tuned.

**Comment by avahe-kellenberger** [comment]
Closed via #42
